Sources/Controllers/2.0/BottomSheet/OABottomSheetTwoButtonsViewController.mm

```diff
@@ -43,28 +43,40 @@ - (void)additionalSetup
 
 - (void) setupButtons
 {
+    [self layoutButtons];
     if (_doneButton.hidden)
```

Comment:
Why do you need to hide the button if it's hidden? Also, why do you call [self layoutButtons] twice (here and in [self hideDoneButton];)? It's better to call it after [self hideDoneButton] and remove it from the  hideDoneButton method.